ui/js/component/transactionList/internal/TransactionListItem.jsx

```diff
@@ -6,12 +6,59 @@ import Link from "component/link";
 import lbryuri from "lbryuri";
 
 class TransactionListItem extends React.PureComponent {
+  abandonClaim() {
+    const {
+      claim_id: claimId,
+      claim_name: name,
+      txid,
+      type,
+      nout,
+    } = this.props.transaction;
+    let msg;
+
+    if (type == "tip") {
+      msg = "This will reduce the committed credits to the URL";
+    } else {
+      msg = "This will reclaim you lbc, back to your account";
+    }
+
+    const abandonData = {
```

Comment:
The only input to revoking a claim should be the transaction id (or the outpoint(s)?). The rest of this data can be reselected rather than passed.

More generally, it is usually a bad idea to create arbitrary data structures to pass multiple values. Even if 4 or 5 values were required here, I'd prefer them be passed individually.

Reply:
Done.